tensorboard/plugins/audio/summary_v2.py

```diff
@@ -43,6 +43,12 @@ def audio(
 ):
     """Write an audio summary.
 
+    Usage Example:
+
+    >>> audio = tf.reshape(tf.linspace(0.0, 50.0, 4 * 8 * 2), (4, 8, 2))
+    >>> tf.summary.audio('k488', audio, 2)
+    <tf.Tensor: shape=(), dtype=bool, numpy=...>
```

Comment:
Could we go a bit deeper into whatever `numpy` is supposed to be instead of going straight to the `...`?